FOUNDATION_CODE_OF_CONDUCT_REQUIREMENTS.md

```diff
@@ -1,28 +1,27 @@
 # Foundation Code of Conduct Requirements
 
 It is important that there be a Code of Conduct (CoC) which
-is documented and enforced for the OpenJS Foundation, 
-hereinafter referred to as the "Foundation", and the
+is documented and enforced for the OpenJS Foundation and the
 projects within it. This document defines the requirements
-for a CoC for the Foundation and the projects which are part
+for a CoC for the foundation and the projects which are part
 of the foundation.
```

Comment:
Optional suggestion to focus it a bit more:

```suggestion
This document defines the requirements for a Code of Conduct (CoC) for
projects within the OpenJS Foundation.
```

Reply:
Can't seem to commit your changes here, @Trott, but I'll include them manually.